test/constants.ts

```diff
@@ -294,5 +297,22 @@ export const DUTCH_LIMIT_ORDER_JSON = {
   startTimeBufferSecs: 45,
   auctionPeriodSecs: 60,
   slippageTolerance: '0.5',
-  orderHash: '0x8859113385dac928f6e064e6d49539fd94cab32687e1a37592ef6f3192948513'
+  orderHash: '0x8859113385dac928f6e064e6d49539fd94cab32687e1a37592ef6f3192948513',
 };
+
+export const GREENLIST_TOKENS: Array<Currency> = [
+  Ether.onChain(ChainId.MAINNET),
+  USDC_MAINNET,
+  DAI_MAINNET,
+  WETH9[ChainId.MAINNET],
+  UNI_MAINNET,
+  WBTC_MAINNET,
+  agEUR_MAINNET,
+  GUSD_MAINNET,
+  LUSD_MAINNET,
+  EUROC_MAINNET,
+  XSGD_MAINNET,
+];
+
+export const GREENLIST_USD_STABLES: Array<Currency> = [USDC_MAINNET, DAI_MAINNET, GUSD_MAINNET, LUSD_MAINNET];
+export const GREENLIST_EUR_STABLES: Array<Currency> = [EUROC_MAINNET, agEUR_MAINNET];
```

Comment:
nit: do we need to make a distinction here? Like can we just do `GREENLIST_STABLES`

Reply:
Right I understand that some pairs will have different fees than others, but these are tests and the actual fee set per pair shouldn't affect the results of the test. Like we also don't have to test every combination of possible fee tokens either, just one from each edge case.

Reply:
Like how in the regular integration tests we basically only swap between USDC, USDT, UNI, WETH, and ETH. This covers popular stablecoins, erc20s (uni), weth, and native currency